tests/test_execution.py

```diff
@@ -251,6 +257,90 @@ def test_task_execution_failure(factory, network, default_dataset_1, worker):
             assert client.get_logs(traintask.key) == logs
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "identifier_1,identifier_2,expectation",
+    [
+        (
+            "name_1",
+            "name_2",
+            does_not_raise(),
+        ),
+        (
+            "same_name",
+            "same_name",
+            pytest.raises(ValueError),
+        ),
+    ],
+)
+def test_custom_testtask(factory, client, default_dataset, worker, identifier_1, identifier_2, expectation):
+    """Test with a test task with several performance output."""
+
+    custom_metric_script = f"""
+import json
+import substratools as tools
+
+@tools.register
+def score(inputs, outputs, task_properties):
+    tools.save_performance(1, outputs['{identifier_1}'])
+    tools.save_performance(2, outputs['{identifier_2}'])
+
+if __name__ == '__main__':
+    tools.execute()
+    """
+
+    # add test data samples / dataset / metric on organization 1
+    spec = factory.create_function(FunctionCategory.simple)
+    function = client.add_function(spec)
+
+    predict_function_spec = factory.create_function(FunctionCategory.predict)
+    predict_function = client.add_function(predict_function_spec)
+
+    # add traintask on organization 2; should execute on organization 2 (dataset located on organization 2)
+    spec = factory.create_traintask(
+        function=function,
+        inputs=default_dataset.train_data_inputs,
+        worker=worker,
+    )
+    traintask = client.add_task(spec)
+    traintask = client.wait(traintask)
+
+    # add testtask; should execute on organization 1 (default_dataset_1 is located on organization 1)
```

Comment:
Same here. Wondering about organisation 1/2 since I don't see where it's defined in this test